src/Acuminator/Acuminator.Utilities/Roslyn/Constants.cs

```diff
@@ -0,0 +1,641 @@
+			internal static readonly string PXDBIntAttribute = "PX.Data.PXDBIntAttribute";
+			internal static readonly string PXDBShortAttribute = "PX.Data.PXDBShortAttribute";
+			internal static readonly string PXDBStringAttribute = "PX.Data.PXDBStringAttribute";
+			internal static readonly string PXDBByteAttribute = "PX.Data.PXDBByteAttribute";
+			internal static readonly string PXDBDecimalAttribute = "PX.Data.PXDBDecimalAttribute";
+
+			#endregion
+
+			#region PXUIFieldAttributeSymbols
+			internal static readonly string PXUIFieldAttribute = "PX.Data.PXUIFieldAttribute";
+			internal static readonly string PXUIFieldAttributeSetVisible = "SetVisible";
+			internal static readonly string PXUIFieldAttributeSetVisibility = "SetVisibility";
+			internal static readonly string PXUIFieldAttributeSetEnabled = "SetEnabled";
+			internal static readonly string PXUIFieldAttributeSetRequired = "SetRequired";
+			internal static readonly string PXUIFieldAttributeSetReadOnly = "SetReadOnly";
+			internal static readonly string PXUIFieldAttributeSetDisplayName = "SetDisplayName";
+			internal static readonly string PXUIFieldAttributeSetNeutralDisplayName = "SetNeutralDisplayName";
+			#endregion
+
+			#region PXFilteredProcessingGraphRule
+			internal static readonly string PXFilteredProcessing = "PX.Data.PXFilteredProcessing`2";
+			#endregion
+
+			#region ViewsWithoutPXViewNameAttributeGraphRule && PXViewNameAttributeViewRule
+			internal static readonly string PXViewNameAttribute = "PX.Data.PXViewNameAttribute";
+			#endregion
+
+			#region PXContext
+			internal static readonly string PXGraphExtension = "PX.Data.PXGraphExtension";
+			internal static readonly string PXCacheExtension = "PX.Data.PXCacheExtension";
+			internal static readonly string PXMappedCacheExtension = "PX.Data.PXMappedCacheExtension";
+			internal static readonly string PXLongOperation = "PX.Data.PXLongOperation";
+			internal static readonly string PXActionCollection = "PX.Data.PXActionCollection";
+			internal static readonly string PXAdapter = "PX.Data.PXAdapter";
+			
+			internal static readonly string IBqlField = "PX.Data.IBqlField";
+			internal static readonly string Constant = "PX.Data.Constant`1";
+			internal static readonly string IPXResultset = "PX.Data.IPXResultset";
+			internal static readonly string PXResult = "PX.Data.PXResult";
+			internal static readonly string PXFieldState = "PX.Data.PXFieldState";
+			internal static readonly string PXAttributeFamilyAttribute = "PX.Data.PXAttributeFamilyAttribute";
+			internal static readonly string IPXLocalizableList = "PX.Data.IPXLocalizableList";
+			internal static readonly string PXConnectionScope = "PX.Data.PXConnectionScope";
+
+			internal static readonly string PXGraphExtensionInitialize = "Initialize";
+			internal static readonly string PXLongOperationStartOperation = "StartOperation";
+
+			internal static readonly string IImplementType = "PX.Common.IImplement`1";
+			#endregion
+
+			#region AttributeSymbol.cs
```

Comment:
I don't think we need .cs endings in region names

Reply:
I agree with @SENya1990.
@metallmonster In this case, it is done intentionally to keep all string literals in one place instead of separating them into several files.

Reply:
@dropsonic what are the benefits of this approach? For me it is difficult to work with a big file full of regions. I don't know, maybe it is Ok for you, but I'd prefer to have several files unified through single namespace. It would improve readability and decrease complexity of the file.

Reply:
@metallmonster because from the user standpoint it should be the same class (`Types` and `TypeNames`) instead of multiple files. It could be achieved by using partial classes, but I don't think it makes any sense because nobody will read this file.

Reply:
Well actually I would also prefer if this file was splitted into multiple partial files - a partial Types class + nested class with constants. I think it will be more structured than regions - it will be easier to add new constant in a right place.

Reply:
But I won't insist on it because imho it isn't too important issue

Reply:
There are too many duplicates and unobvious nested types for constants which were part of legacy and etc. TypeNames, for example. We could either keep them as they are and postpone refactoring - in this case in my opinion one file is actually better. Or we can regroup this constants in scope of this pull request and in this case I would prefer a few big logically groups of constants each kept in a separate file.

Reply:
I agree with @SENya1990 , if we don't want to refactor this file it is OK to keep it like this. But later we need to do something with it.